bin/sl-deploy.js

```diff
@@ -144,11 +150,20 @@ if (!local) {
 function exit(err, service) {
   var svc = service ? (service.name || service.id) : serviceName;
   if (err) {
-    console.error('Failed to deploy `%s` as `%s` to `%s` via `%s`',
-                  branchOrPack, svc, baseURL, exit.url);
```

Comment:
I am trying to find the globalize docs for this `{ key}, {key: value}` syntax, where are they?

Some tools do:

`glb.error('this {$1} is a {$2}`, 'statue', 'dog')`... basically keys prefixed with a `$` are positional args. its more concise for some uses. Does globalize support this? If not, maybe we can suggest it.